src/memory.rs

```diff
@@ -82,6 +82,9 @@ pub struct MemoryQuantity {
 ///   supports up to Gibibytes while K8S quantities can be expressed in Exbibytes.
 /// - the heap size has a non-zero value.
 /// Fails if it can't enforce the above restrictions.
+#[deprecated(
+    note = "use \"-Xmx\" + MemoryQuantity::try_from(quantity).scale_to(unit).format_for_java()"
+)]
```

Comment:
I would add the version (if the new tag will be 0.33.0). Did not check clippy for this. Same for `to_java_heap_value`
```suggestion
#[deprecated(
    since= "0.33.0",
    note = "use \"-Xmx\" + MemoryQuantity::try_from(quantity).scale_to(unit).format_for_java()"
)]
```

Reply:
done